oct/ansible/oct/playbooks/prepare/golang.yml

```diff
@@ -66,7 +66,7 @@
         - 'golang.org/x/tools/cmd/goimports'
         - 'github.com/tools/godep'
         - 'github.com/golang/lint/golint'
-        - 'github.com/josephspurrier/goversioninfo'
+#        - 'github.com/josephspurrier/goversioninfo'
```

Comment:
Commented out this module install due to the following failure:
```
failed: [172.18.5.150] (item=github.com/josephspurrier/goversioninfo) => {
    "changed": true, 
    "cmd": [
        "go", 
        "get", 
        "github.com/josephspurrier/goversioninfo"
    ], 
    "delta": "0:00:01.004064", 
    "end": "2020-11-11 20:01:28.777890", 
    "failed": true, 
    "generated_timestamp": "2020-11-11 15:01:28.797841", 
    "item": "github.com/josephspurrier/goversioninfo", 
    "rc": 2, 
    "start": "2020-11-11 20:01:27.773826", 
    "stderr": [
        "# github.com/akavel/rsrc/coff", 
        "/data/src/github.com/akavel/rsrc/coff/coff.go:169:18: undefined: pe.IMAGE_FILE_MACHINE_ARMNT", 
        "/data/src/github.com/akavel/rsrc/coff/coff.go:172:18: undefined: pe.IMAGE_FILE_MACHINE_ARM64", 
        "/data/src/github.com/akavel/rsrc/coff/coff.go:265:7: undefined: pe.IMAGE_FILE_MACHINE_ARMNT", 
        "/data/src/github.com/akavel/rsrc/coff/coff.go:267:7: undefined: pe.IMAGE_FILE_MACHINE_ARM64"
    ], 
    "stdout": [], 
    "warnings": []
}
```
Will come back to fix this if necessary.